.github/workflows/ci.yml

```diff
@@ -10,40 +10,37 @@ concurrency:
 
 jobs:
   tests:
-    name: ${{ matrix.PYTHON_VERSION }} ${{ matrix.POLARS_VERSION }}
+    name: Unit tests
     timeout-minutes: 15
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        include:
-          - { PYTHON_VERSION: 'python=3.9', POLARS_VERSION: 'polars=0.14.28' }
-          - { PYTHON_VERSION: 'python=3.9', POLARS_VERSION: 'polars=0.15' }
-          - { PYTHON_VERSION: 'python=3.9', POLARS_VERSION: 'polars=0.16' }
-          - { PYTHON_VERSION: 'python=3.9', POLARS_VERSION: 'polars=0.17' }
-          - { PYTHON_VERSION: 'python=3.9', POLARS_VERSION: 'polars=0.18' }
-          - { PYTHON_VERSION: 'python=3.9', POLARS_VERSION: 'polars=0.19' }
-          - { PYTHON_VERSION: 'python=3.9', POLARS_VERSION: 'polars=0.20' }
-          - { PYTHON_VERSION: 'python=3.10', POLARS_VERSION: '' }
-          - { PYTHON_VERSION: 'python=3.11', POLARS_VERSION: '' }
-          - { PYTHON_VERSION: 'python=3.12', POLARS_VERSION: '' }
+        env:
+          - pl014
+          - pl015
+          - pl016
+          - pl017
+          - pl018
+          - pl019
+          - pl020
+          - py39
+          - py310
+          - py311
+          - py312
     steps:
       - uses: actions/checkout@v4
-      - uses: prefix-dev/setup-pixi@v0.4.3
+      - uses: prefix-dev/setup-pixi@c2a7a2709894350902c4ef78eb3c38f5ae15c7f4
         with:
-          run-install: false
-      - name: Install dependencies
-        # TODO: make prettier once there are feature flags
-        # https://github.com/prefix-dev/pixi/issues/239
+          environments: ${{ matrix.env }}
+      - name: Install repository
         run: |
-          pixi add ${{ matrix.PYTHON_VERSION }} ${{ matrix.POLARS_VERSION }}
-          pixi install
-          pixi run postinstall
+          pixi run -e ${{ matrix.env }} postinstall
```

Comment:
Is the install implicit here?

Reply:
Yes, but the install is done in `setup-pixi` with the `environments:` key.

Reply:
Ah i see. Neat!